standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HMSHandler.java

```diff
@@ -7420,6 +7420,97 @@ public boolean delete_partition_column_statistics(String dbName, String tableNam
     return ret;
   }
 
+
+  public boolean delete_column_statistics_req(DeleteColumnStatisticsRequest req) throws TException {
+    boolean isDeletingTableLevel = req.getPart_name() == null ? true : false;
+    // partioned or not
+    // isPartitoned
+    String dbName = req.getDb_name();
+    String tableName = req.getTbl_name();
+    List<String> colNames = req.getCol_names();
+    String partName = req.getPart_name();
+    String engine = req.getEngine();
+    dbName = dbName.toLowerCase();
+    String[] parsedDbName = parseDbName(dbName, conf);
+    tableName = tableName.toLowerCase();
+
+    if (colNames != null) {
+      for (String colName : colNames) {
+        colName = colName.toLowerCase();
+        if (isDeletingTableLevel) {
+          startFunction("delete_column_statistics_by_table", ": table=" +
```

Comment:
No need to call multiple `startFunction`s, we can just put the colNames into the logs, such as:
```
 startFunction("delete_column_statistics_req", ": table=" +
                  TableName.getQualified(parsedDbName[CAT_NAME], parsedDbName[DB_NAME], tableName) +
                  " partition=" + partName + " column=" + colNames + " part_name=" + partName);
```